plugins/entando-plugin-jpkiebpm/src/main/java/com/agiletec/aps/system/services/page/KiePageManager.java

```diff
@@ -48,7 +48,7 @@ public void removeWidget(String pageCode, int pos) throws ApsSystemException {
         try {
             Widget widget = this.getDraftPage(pageCode).getWidgets()[pos];
             super.removeWidget(pageCode, pos);
-            String stringId = widget != null ? widget.getConfig().getProperty(KieBpmSystemConstants.WIDGET_PARAM_DATA_UX_ID) : null;
+            String stringId = widget != null && widget.getConfig() != null ? widget.getConfig().getProperty(KieBpmSystemConstants.WIDGET_PARAM_DATA_UX_ID) : null;

```

Comment:
Change this to an if statement with a setter. Too long for trinary operator. Tough to tell what is happening here.